hs-abci-examples/nameservice/src/Nameservice/Protogen.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE MagicHash #-}
+
+-- @NOTE: ^ possibly the only language extension needed from tutorial
+-- Seems to be the only requirement for generating the .proto string
+
+module Nameservice.Protogen where
```

Comment:
This should be moved to a separate directory and labeled as an `executable` in package.yaml